files/arith4.cpp

```diff
@@ -1,7 +1,8 @@
 #include <iostream>
 
 int main() {
-    int a, b;
+    int a;
```

Comment:
Selber Kommentar bzgl. undefiniertes Verhalten wie im issue. Ist zwar hier wsl kein Problem aber man will ja gar nicht erst schlechtes Verhalten antrainieren

```suggestion
    int a{};
```